lib/+mwbs/+Visualizers/+robotVisualizer/initRobotVisualizer.m

```diff
@@ -1,3 +1,3 @@
 %% configuration for the matlab iDyntree visualizer
 
-confVisualizer.modelPath = mwbs.getModelPathFromFileNameAndYarpFinder(confVisualizer.fileName);
+[confVisualizer.modelPath,confVisualizer.fileName] = mwbs.getModelPathFromFileNameAndYarpFinder(confVisualizer.fileName);
```

Comment:
Let me check if I understood correctly: before this line `confVisualizer.fileName` contains an absolute or relative path to the file (so not just the name), whil after it just contains the name of the file, right? Could it make sense to use two different variables to avoid confusion?

Reply:
If we can't change the name of variable for backward compatibility, I would add a line of comment: remember, code is read many more times then it is written.

Reply:
> Let me check if I understood correctly: before this line confVisualizer.fileName contains an absolute or relative path to the file (so not just the name), whil after it just contains the name of the file, right?

Yes, exactly.

> Could it make sense to use two different variables to avoid confusion?

Nice suggestion. Previously, we use "fileName" because that variable must be only the file name. But, now that we can also pass the model path to this variable, it has no sense to call it "fileName" anymore. So, I edited names. Now we have three parameters:

- `fileName`: only the name of the model file,
- `modelPath`: Only the path of the model file.
- `UrdfFile`: that can be only the name of the file or the name plus the path

Reply:
Is this backward compatible, or users now need to update their init scripts?

Reply:
I edited the init files in fb0c38f4a611114351e7b16fb4ca63312d0a7a1e, to provide backward compatibility. Now, the users do not need to change their init file. So, now if "UrdfFile" is not provided by the user, we check if "fileName" is provided, and we substitute "fileName" into "UrdfFile".